roads.mss

```diff
@@ -3106,3 +3106,114 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
     }
   }
 }
+#railways-text-name {
+  /* Mostly started from z17. */
+  [railway = 'rail'],
+  [railway = 'subway'],
+  [railway = 'narrow_gauge'],
+  [railway = 'light_rail'],
+  [railway = 'preserved'],
+  [railway = 'funicular'],
+  [railway = 'monorail'],
+  [railway = 'turntable'],
```

Comment:
A turntable is something like the railway equivalent of a parking lane crossing or so. I don't think any of them can have a useful name.